websites/A/Anime Genzai/presence.ts

```diff
@@ -0,0 +1,226 @@
+const presence = new Presence({
+		clientId: "1071365131616915517",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+async function getStrings() {
+	return presence.getStrings(
+		{
+			anime: "general.anime",
+			viewHome: "general.viewHome",
+			viewing: "general.viewing",
+			search: "general.search",
+			searchFor: "general.searchFor",
+			play: "general.watchingVid",
+			pause: "general.paused",
+			searchSomething: "general.searchSomething",
+			watchingMovie: "general.watchingMovie",
+			watchVideoButton: "general.buttonWatchVideo",
+			buttonViewPage: "general.buttonViewPage",
+			viewPage: "general.viewPage",
+			viewMovie: "general.viewMovie",
+			chapter: "general.chapter",
+			viewThread: "general.viewThread",
+			viewProfile: "general.viewProfile",
+		},
+		await presence.getSetting<string>("lang").catch(() => "es")
+	);
+}
+
+enum Assets {
+	Logo = "https://i.imgur.com/dtmYEl8.png",
+	Reading = "https://i.imgur.com/wPUmqu5.png",
+	Searching = "https://i.imgur.com/UhPgTRn.png",
+	Viewing = "https://i.imgur.com/RMg2Qgg.png",
+	Play = "https://i.imgur.com/juDzimk.png",
+	Pause = "https://i.imgur.com/GbAMPHI.png",
+}
+
+let video = {
+		duration: 0,
+		currentTime: 0,
+		paused: true,
+	},
+	strings: Awaited<ReturnType<typeof getStrings>>;
```

Comment:
This is a better way to do this (1/3):
```suggestion
strings: Awaited<ReturnType<typeof getStrings>>,
	oldLang: string = null;
```